accProcess.py

```diff
@@ -203,6 +203,26 @@ def main():
                             metavar='True/False', default=False, type=str2bool,
                             help="""Save intensity distribution
                              (default : %(default)s)""")
+    parser.add_argument('--psd',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate power spectral density for 24 hour 
+                                    circadian period
+                             (default : %(default)s)""")
+    parser.add_argument('--fourierFrequency',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate dominant frequency of sleep
+                             (default : %(default)s)""")
+    parser.add_argument('--m10l5',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate relative amplitude of most and 
+                                    least active acceleration periods
```

Comment:
for circadian rhythm analysis

Reply:
Resolved by 9cf6abf1c266b0d55e154fe2ba73f73e970d1472